ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Validators/Abstract/BaseTaskDTOValidator.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace ExadelTimeTrackingSystem.BusinessLogic.Validators.Abstract
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using ExadelTimeTrackingSystem.BusinessLogic.DTOs;
+    using ExadelTimeTrackingSystem.BusinessLogic.Services.Abstract;
+    using FluentValidation;
+
+    public abstract class BaseTaskDTOValidator<TDTO> : AbstractValidator<TDTO>
+        where TDTO : CreateTaskDTO
+    {
+        private readonly IUserService _userService;
+        private readonly IProjectService _projectService;
+
+        public BaseTaskDTOValidator(IUserService userService, IProjectService projectService)
+        {
+            _userService = userService;
+            _projectService = projectService;
+            ConfigureRules();
+        }
+
+        private void ConfigureRules()
+        {
+            RuleFor(t => t.HoursSpent)
+                .InclusiveBetween(0, 1000);
```

Comment:
Task is tracked for the day. How many max hours can be spent for a day? :) And exclude 0 from the range, we should not allow tracking task for which user worked 0 hours